### ISSUE TITLE
Add Solutions in Golang Language

### ISSUE THREAD

**Issue by sagarchoudhary96** [sagarchoudhary96/30-Days-Of-Code#5]
Solutions are welcomed for each problem in golang. Please check **contributing Guidelines** before starting.  You can register for **Hacktoberfest** [here](https://hacktoberfest.digitalocean.com/) to receive **free swags** once you raise 4 PR.

- [x] Day 0 **Completed**
- [x] Day 1 **Completed**
- [x] Day 2  **Completed**
- [x] Day 3  **Completed**
- [x] Day 4  **Completed**
- [x] Day 5  **Completed**
- [x] Day 6  **Completed**
- [x] Day 7  **Completed**
- [x] Day 8 **Completed**
- [x] Day 9  **Completed**
- [x] Day 10  **Completed**
- [x] Day 11 **Completed**
- [x] Day 12 Cannot be solved in Go language on hackerrank
- [x] Day 13 Cannot be solved in Go language on hackerrank
- [x] Day 14 Cannot be solved in Go language on hackerrank
- [x] Day 15 Cannot be solved in Go language on hackerrank
- [x] Day 16 Cannot be solved in Go language on hackerrank
- [x] Day 17 Cannot be solved in Go language on hackerrank
- [x] Day 18 Cannot be solved in Go language on hackerrank
- [x] Day 19 Cannot be solved in Go language on hackerrank
- [x] Day 20 **Completed**
- [x] Day 21 Cannot be solved in Go language on hackerrank
- [x] Day 22 Cannot be solved in Go language on hackerrank
- [x] Day 23 Cannot be solved in Go language on hackerrank
- [x] Day 24 Cannot be solved in Go language on hackerrank
- [x] Day 25  **Completed**
- [x] Day 26 **Completed**
- [x] Day 27 Cannot be solved in Go language on hackerrank
- [X] Day 28 **Completed**
- [x] Day 29 **Completed**

**Comment by Jolly1234** [comment]
Hi, @sagarchoudhary96 I would like to work on this.

**Comment by sagarchoudhary96** [comment]
@Jolly1234 , mention the day numbers for which you would like to work on first. then you can start working on it and  make a PR for it.

**Comment by Kausam** [comment]
@sagarchoudhary96, I would like to work on problems part of Day 0 to Day 15.

**Comment by sagarchoudhary96** [comment]
@Kausam, you can pick max 4 at once and send a PR for it so that it's easy for me to review too and others can also get chance to contribute. once one apr is merged then you can pick other 4.

**Comment by Kausam** [comment]
Okay, I will take up the first 4 problems. That is problems from Day 0 to Day 3.

**Comment by sagarchoudhary96** [comment]
okay @Kausam , go ahead👍

**Comment by garvita-jain** [comment]
@sagarchoudhary96 I'd like to work on Day 9 and 10 problems. Can you please assign those to me.

**Comment by sagarchoudhary96** [comment]
sure @garvita-jain

**Comment by Jolly1234** [comment]
Hi Sagar,
 Anything is fine.. day 4,5,6,7 I can pick

**Comment by Jolly1234** [comment]
Also, Sagar, I was thinking of updating readme.md like right now it contains only the solutions may be a link for questions would be better ( just a suggestion if u are fine with it)

**Comment by sagarchoudhary96** [comment]
hey @Jolly1234 , sure go ahead you can update Readme too. I will Create a new issue for it and assign to you.

**Comment by sagarchoudhary96** [comment]
hi @garvita-jain @Kausam , request you to start sending solution for the question solutions you have implemented one by one.

**Comment by vigneshashokan** [comment]
hi @sagarchoudhary96 , may I have 8, 11, 12 and 13, please?

**Comment by sagarchoudhary96** [comment]
Sure go ahead.

**Comment by vigneshashokan** [comment]
@sagarchoudhary96 , getting this error while trying to push the branch. Are you supposed to grant me access?
```
ERROR: Permission to sagarchoudhary96/30-Days-Of-Code.git denied to vigneshashokan.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

**Comment by sagarchoudhary96** [comment]
You cant directly push to my repo. You need to fork the repo then make changes in forked repo commit there and maie a pull request from forked repo to my repo.

**Comment by kmalik31** [comment]
I'll work on Day14

**Comment by sagarchoudhary96** [comment]
@kmalik31 sure go ahead 👍

**Comment by gmgalvan** [comment]
hi @sagarchoudhary96 , can i solve day 15?

**Comment by sagarchoudhary96** [comment]
@gmgalvan sure go ahead, will assign it to you

**Comment by sagarchoudhary96** [comment]
@kmalik31 any updates on the progress?

**Comment by sagarchoudhary96** [comment]
@sagarchoudhary96 @vigneshashokan @kmalik31  can you guys update about your progress. if u are not working on it , let me know  will open them up so that anyone else can pick it up.

**Comment by vigneshashokan** [comment]
Sorry about the delay, @sagarchoudhary96 .. I got held up with something. I'm working on it and I'll have the PRs ready over the weekend.

**Comment by sagarchoudhary96** [comment]
sure @vigneshashokan  looking forward for your PR's :)

**Comment by vigneshashokan** [comment]
Quick question @sagarchoudhary96 , there is no Golang option available to submit in Hackerrank. If I just have a working solution for Day 12 and 13 is it fine? or is there a place where you want me to check my solution for test cases before I can create a PR?

**Comment by sagarchoudhary96** [comment]
it would be great if you can let me know which question is not available to be solved in golang on hackerrank, you can skip that one and work on the one which is available.

**Comment by sagarchoudhary96** [comment]
@vigneshashokan if both are not available you can pick from other available question and let mee know will assign them to you.

**Comment by Kausam** [comment]
@sagarchoudhary96 unfortunately day14 problem doesn't accept any solutions in Golang :(

**Comment by sagarchoudhary96** [comment]
@Kausam  you can look in other open question we are about to finish it, i think only 28,29 is available on hackerrank in golang, can you check and update once and also if you would like to take available question.

**Comment by Kausam** [comment]
Sure. Looks like Day 29 is already done. I'd be happy to take up Day28

**Comment by sagarchoudhary96** [comment]
@Kausam sure go ahead, waiting for your PR :)

**Comment by sagarchoudhary96** [comment]
we have finally finished this too. thanks to everyone for contributing to this issue 🎉😄
